addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -120,6 +129,20 @@ class ScEnvironmentCard extends React.Component {
     );
   }
 
+  renderWarning(env) {
+    if (isAppStreamEnabled && !env.isAppStreamConfigured && env.state.canTerminate) {
+      return (
+        <Message
+          icon="warning"
+          header="Non-AppStream workspace found"
+          content="Please terminate this workspace to avoid AppStream configuration issues for its hosting account"
```

Comment:
Thoughts on this error message? 
`Please terminate this workspace as soon as possible. This workspace is not supported by AppStream.`

It makes it less ambiguous, because people might ask what `AppStream configuration issues` when they see the message.